src/audio/mixer.c

```diff
@@ -366,8 +366,7 @@ static int mixer_copy(struct comp_dev *dev)
 		 * generating silence until at least one of the
 		 * sources start to have data available (frames!=0).
 		 */
```

Comment:
should the comment be updated too? TBH I don't understand how this is working. AFAIK at least in a typical example of pipelines A and B as sources to mixer on pipeline C, with output to a DAI, currently the whole pipeline C will stay active infinitely and play back silence if, say, A is stopped and B is paused. How is this PR changing that?

Reply:
@lyakh the comment looks OK to me even with the change. The change that this PR addresses is when all the sources are paused or when there's no data available yet from one of the active sources. In that case, why should we fill up the entire free space in the output buffer? We should still be producing only period bytes right?

Reply:
@ranj063 ah, ok, I see now, I misunderstood the change. Yes, I think you're right, thanks for the explanation